core/database/src/main/java/com/gkp/core/database/entity/AgendaItemEntity.kt

```diff
@@ -0,0 +1,27 @@
+package com.gkp.core.database.entity
+
+import android.annotation.SuppressLint
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.gkp.agenda.domain.util.toLocalDateTime
+import java.time.LocalDate
+import java.time.ZoneId
+
+@Entity
+data class AgendaItemEntity(
+    @PrimaryKey(autoGenerate = false)
+    val id: String,
+    val title: String,
+    val description: String?,
+    val time: Long,
+    val to: Long? = null,
+    val remindAt: Long,
+    val isDone: Boolean? = null,
+    val photoUrls : String? = null,
+    val date: Long = time.toLocalDateTime().toLocalDate().toMillis()
+)
```

Comment:
What do you think about having a different entity for each agenda item instead? If you think about it, sure, a task, a reminder, and an event are all agenda items, but does that mean that you have to store them in the same DB table? As you can see, you already had to make some fields nullable, which are mandatory for other agenda items (take the `description` field, which is only optional for an Event).

Each item has its own requirements that can change independently of each other. An excellent way to model that is to have a table for each agenda item type.